plugins/cache/memcached.py

```diff
@@ -180,13 +180,7 @@ def __init__(self, *args, **kwargs):
             self._timeout = self.get_option('_timeout')
             self._prefix = self.get_option('_prefix')
         except KeyError:
-            display.deprecated('Rather than importing CacheModules directly, '
```

Comment:
Looks like Ansible 2.9 still needs this.

Maybe we should add an Ansible(-base/core) version check here: if the Ansible version is < 2.10, use the old logic (and remove deprecation), and if the Ansible-base/core version is >= 2.10, raise an `AnsibleError`.

Reply:
Implemented in 46454d13ac68f94cfe74a00ff870d56e0b865965.

Reply:
Should we create an issue for this in the repo, so we don't forget about it?